baggage/baggage.go

```diff
@@ -155,6 +160,9 @@ func (p Property) Value() (string, bool) {
 
 // String encodes Property into a header string compliant with the W3C Baggage
 // specification.
+// It would return empty string if the key is invalid with the W3C Baggage
+// specification. This could happen for a UTF-8 key, as it may contain
+// invalid characters.
```

Comment:
Is it not repetitive?

```suggestion
// specification.
```